### DISCUSSION TITLE
Shortcut keys:  Subtract key (-) in text field causes Zoom

### DISCUSSION THREAD

**Discussion by MisterABK** [opentoonz/opentoonz#6794] [General]
Hitting the subtract key zooms out (which seems odd, because hitting + doesn't zoom in). But the issue I'm running into is that typing - in a text field (such as when scaling a raster selection) often causes OpenToonz to zoom out rather than type - into the box. 

I should add that this is an intermittent problem. After 2 or 3 tries I will be able to type a minus sign, and after that the problem may not reoccur for a short while.

**Comment by artisteacher** [comment]
I haven’t run into the issue with text fields myself. 

I have had a similar shortcut issue on my laptop #1327. I don’t have a numberpad so my + key is also my = key.  I’ve changed the shortcut to the =/+ key but don’t really use it. I actually end up using the trackpad to zoom in/out as it is more intuitive and I don’t like the way the shortcuts zoom in/out. To change the shortcut, you can go to  File -> Configure Shortcuts.

**Comment by MisterABK** [comment]
Ah, yes, I'm using a mini-keyboard with + and = on the same key.

**Comment by DarrenTAnims** [comment]
I get a similar thing to the minus key issue in that, if I'm using my tablet, it's easy for the pen to move (natural hand movements, not ghostly) and if you go too far, the text box loses focus and then your keys will do whatever the shortcut does, so pressing minus will zoom out and pressing a number can change the selected brush.

I get really frustrated with the textbox controls because of this and also when you first click in them the text is highlighted and I've got a habit, from using other software, that the text in textboxes isn't selected by default, so instead of just clicking into it, I click and drag, which unselects the text first. And then if my pen moves too far, I lose focus, start again and often click n drag and get the same outcome.

**Comment by MisterABK** [comment]
I try to select the number value and end up deselecting it almost every time too. The wacom pen causing the field to lose focus trips me up too. Perhaps that's a factor with my initial problem?

**Comment by gab3d** [comment]
- to me de auto selection of the numbers is a good thing, because with just one click (the initial one to enter the field) you can already start changing the values.

- the lose of focus is indeed a usability problem IMO. 
perhaps the bounding pixels inside which the focus is maintained could be increased, to make it less probable to lose focus accidentally?

**Comment by RodneyBaker** [comment]
Recommend reviewing this issue and closing as appropriate.
